configure.ac

```diff
@@ -616,9 +616,9 @@ AC_DEFINE_UNQUOTED(FALLBACK_HOSTNAME, ["$FALLBACK_HOSTNAME"], [The hostname used
 
 AC_ARG_WITH(default-hierarchy,
         AS_HELP_STRING([--with-default-hierarchy=MODE],
-                [default cgroup hierarchy, defaults to "legacy"]),
+                [default cgroup hierarchy, defaults to "unified"]),
         [DEFAULT_HIERARCHY="$withval"],
-        [DEFAULT_HIERARCHY="legacy"])
+        [DEFAULT_HIERARCHY="unified"])
```

Comment:
CentOS/Ubuntu Xenial don't support unified hierarchy at all. All testbeds fail to boot with:
```
[!!!!!!] Failed to mount cgroup2 at /sys/fs/cgroup: No such device
[    1.818148] systemd[1]: Freezing execution.
```
We have already discussed that: https://github.com/systemd/systemd/pull/4368#issuecomment-254701160
But I'm not sure what is the result of that discussion